### ISSUE TITLE
URL within link text identified as an error

### ISSUE THREAD

**Issue by treyhunner** [rvben/rumdl#48]
This file:

```md
Also don't forget that the next time you need to figure out which `datetime` format you need, **[use the strptime tool at https://pym.dev/strptime](https://www.pythonmorsels.com/strptime/)**!
```

Has a linting error identified due to the https://pym.dev/strptime URL within the link text.

```bash
$ uv tool run rumdl check --disable MD013,MD041 file17.md
file17.md:1:123: [MD034] URL without angle brackets or link formatting [*]

Issues: Found 1 issues in 1 file (30ms)
Run with `--fix` to automatically fix 1 of the 1 issues
```

Since the URL is within link text, this doesn't seem like it should be an error.

I may be wrong on this one. If you feel this *should* be an error, I'd like to know what would you recommend replacing this link with. Thanks!

**Comment by rvben** [comment]
Interesting one! In a case like this it's probably wise to look at user intent. Having it there means that it is at least not a bare url (like the rule is supposed to trigger on). If we would decide that this is undesirable, it should probably be in a different rule.

Any way, this was a false positive and should be fixed in the 0.0.116 today.

Thanks again.
